apps/style/applab/style.scss

```diff
@@ -500,3 +502,12 @@ button.arrow:focus {
   background-size: 16px 16px;
   background-position: 2px 0px;
 }
+
+.design-mode-hidden {
```

Comment:
Thanks for the tip. My css fu is not all that strong, so I'm always happen to learn how to do things better :)